rust-src/concordium_base/src/id/sigma_protocols/dlog.rs

```diff
@@ -55,28 +52,32 @@ impl<C: Curve> SigmaProtocol for Dlog<C> {
         Some((randomised_point, rand_scalar))
```

Comment:
Also here: maybe we can make the code clearer by renaming `randomised_point` to `commit_message` (although `randomised_point` is also fine)